sdk/search/Azure.Search.Documents/src/SearchClient.cs

```diff
@@ -1663,5 +1566,484 @@ public class SearchClient
             }
         }
         #endregion IndexDocuments
+
+        #region Index Documents Conveniences
+        /// <summary>
+        /// Upload documents to the index as a batch.
+        /// </summary>
+        /// <typeparam name="T">
+        /// The .NET type that maps to the index schema. Instances of this type
+        /// can be retrieved as documents from the index.
+        /// </typeparam>
+        /// <param name="documents">The documents to upload.</param>
+        /// <param name="options">
+        /// Options that allow specifying document indexing behavior.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate notifications
+        /// that the operation should be canceled.
+        /// </param>
+        /// <returns>
+        /// Response containing the status of operations for all actions in the
+        /// batch of actions.
+        /// </returns>
+        /// <exception cref="RequestFailedException">
+        /// Thrown when a failure is returned by the Search Service.
+        /// </exception>
+        /// <remarks>
+        /// <para>
+        /// The generic overloads of the UploadDocuments and UploadDocumentsAsync
+        /// methods support mapping of search field types to .NET types via the
+        /// type parameter T. See
+        /// <see cref="GetDocumentAsync{T}(string, GetDocumentOptions, CancellationToken)"/>
+        /// for more details on the type mapping.
+        /// </para>
+        /// <para>
+        /// By default, an exception will only be thrown if the entire request
+        /// fails.  Individual failures are described in the
+        /// <see cref="IndexDocumentsResult.Results"/> collection.  You can set
+        /// <see cref="IndexDocumentsOptions.ThrowOnAnyError"/> if you want
+        /// exceptions thrown on partial failure.
+        /// </para>
+        /// </remarks>
+        [ForwardsClientCalls]
```

Comment:
I thought we weren't supposed to use these anymore. @pakrym had a Roslyn analyzer that fixed these up. Could probably wait till preview 5, though.

Reply:
...or is this thin enough it doesn't matter in this case?

Reply:
I've got a Preview 5 bug to figure the rest of those our for SearchClient.  I'm going to leave them as-is for the current Preview, but I'll dig into the right way to fix these.